### PR TITLE
gmailctl 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by browniebroke**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gmailctl.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Gmailctl < Formula
+  desc "Declarative configuration for Gmail filters"
+  homepage "https://github.com/mbrt/gmailctl"
+  url "https://github.com/mbrt/gmailctl/archive/v0.6.0.tar.gz"
+  sha256 "e188d5d18ac84ca86c94fb2b90d219ae3bea1e1ddee1966fae904c93bf27f233"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w -X main.version=#{version}",
```
Comment (alebcay) [inline Formula/gmailctl.rb:10]: You should use `*std_go_args` here, which will make things a bit cleaner.
```
system "go", "build", "-ldflags", "-s -w -X main.version=#{version}",
       *std_go_args, "cmd/gmailctl/main.go"
```

For now, `*std_go_args` contains `"-trimpath", "-o", bin/name`.

Comment (browniebroke) [reply Formula/gmailctl.rb]: Nice, thanks for the suggestion.

**Comment by browniebroke** [comment]
Do I need to worry about adding the section with SHAs of each bottle or is it automated by a GitHub action?

**Comment by alebcay** [comment]
The bottles' SHA checksums will be added and updated automatically when PRs are merged.

**Comment by chenrui333** [comment]
Rebased to the master for the new CI flow.

**Comment by chenrui333** [comment]
@browniebroke Thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by alebcay** [comment]
@chenrui333, did you mean to merge this?

**Comment by jasisk** [comment]
> @chenrui333, did you mean to merge this?

Two of the three builds were failing the other day (but have since cleared)—perhaps that was the hold-up?

**Comment by alebcay** [comment]
I ask since the "Without awesome contributors like you..." spiel is usually added at merge time.

**Comment by chenrui333** [comment]
yeah, it is ready to merge.

Sorry being little bit slow this week as overwhelmed by the work.

**Comment by issyl0** [comment]
Thanks again, @browniebroke!

**Pull request closed without merging**